packages/plugins/package.json

```diff
@@ -30,7 +30,8 @@
     "lint:ts:check": "eslint . --max-warnings=0",
     "pre-commit": "lint-staged",
     "yalc:publish": "yarn build && yalc publish",
-    "sync:rpc": "infisical export --path='/RPC' --env=dev >> .env"
+    "sync:rpc": "infisical export --path='/RPC' --env=dev >> .env",
+    "write-version": "npx ts-node script/write-version.ts > src/cli/version.ts"
```

Comment:
This script writes the sphinx plugin version constant. It works the same as the `contracts` package script we already have.